src/composables/useNotifications.ts

```diff
@@ -0,0 +1,23 @@
+import { reactive, toRefs } from 'vue';
+
+interface Notification {
+  message: string;
+  type: string;
+  timestamp: number;
+}
+
+const state = reactive({
```

Comment:
Here its better to use only `const items = ref([]);` as we need only a single item